### ISSUE TITLE
Word, static member function copy

### ISSUE THREAD

**Issue by arynaq** [hioa-cs/cpp_v2015#10]
Hei, usikker på hva du mener med static member function copy(Word& w) som skal kopiere innholdet i w "hit". "hit" eksisterer ikke vel når den funksjonen er statisk?

**Comment by alfred-bratterud** [comment]
Jo, du har helt rett - den må enten ta to parametre eller ikke være static. Skal fikse.
